src/main/resources/swagger.api/schemas/user.yaml

```diff
@@ -149,3 +149,19 @@ Tags:
       items:
         type: string
   additionalProperties: false
+
+UserCollection:
```

Comment:
We already have java class UserCollection, please delete it because we have now this yaml file